<!DOCTYPE HTML>
<html>
	<head>
		<title>Serrano-Pozo Lab</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no"/>
		<link rel="shortcut icon" type="image/svg" href="images/favicon.ico"/>
		<link rel="stylesheet" href="assets/css/main.css"/>
		<link rel="stylesheet" href="assets/css/custom.css"/>
		<link rel="stylesheet" href="assets/css/bib-publication-list.css" type="text/css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css"/></noscript>
	</head>
	<body class="is-preload">

		<!-- Page Wrapper -->
			<div id="page-wrapper">

				<!-- Header -->
					<header id="header">
						<h1><a href="https://www.serranopozolab.org">Serrano-Pozo Lab</a></h1>
						<nav>
							<a href="#menu">Menu</a>
						</nav>
					</header>

				<!-- Menu -->
					<nav id="menu">
						<div class="inner">
							<h2>Menu</h2>
							<ul class="links">
								<li><a href="https://www.serranopozolab.org">Home</a></li>
								<li><a href="https://www.serranopozolab.org/research">Research</a></li>
								<li><a href="https://www.serranopozolab.org/team">Team</a></li>
								<li><a href="https://www.serranopozolab.org/location">Location</a></li>
								<li><a href="https://www.serranopozolab.org/publications">Publications</a></li>
								<li><a href="#contact">Contact</a></li>
							</ul>
							<a href="#" class="close">Close</a>
						</div>
					</nav>

				<!-- Wrapper -->
					<section id="wrapper">
						<header>
							<div class="inner">
								<h2>Publications</h2>
								<p>Recent papers and featured projects</p>
							</div>
						</header>

						<!-- Content -->
							<div class="wrapper">
								<div class="inner">

									<!--<h3 class="major">About</h3>
									<p>The Serrano-Pozo Lab investigates the role of astrocytes and microglia in Alzheimer's disease using wet and dry lab approaches.</p>

									<h3 class="major">Publications</h3>
									<p>Please see our most recent work below.</p> -->

									
									<table id="pubTable" class="display"></table>
									<pre id="bibtex">


@article{qian_neuropathology-independent_2023,
	title = {Neuropathology-{Independent} {Association} {Between} {APOE} {Genotype} and {Cognitive} {Decline} {Rate} in the {Normal} {Aging}-{Early} {Alzheimer} {Continuum}},
	volume = {9},
	copyright = {Copyright © 2023 The Author(s). Published by Wolters Kluwer Health, Inc. on behalf of the American Academy of Neurology.. This is an open access article distributed under the terms of the Creative Commons Attribution-NonCommercial-NoDerivatives License 4.0 (CC BY-NC-ND), which permits downloading and sharing the work provided it is properly cited. The work cannot be changed in any way or used commercially without permission from the journal.},
	issn = {2376-7839},
	url = {https://ng.neurology.org/content/9/1/e200055},
	doi = {10.1212/NXG.0000000000200055},
	abstract = {Background and Objectives We previously found that the APOE genotype affects the rate of cognitive decline in mild-to-moderate Alzheimer disease (AD) dementia independently of its effects on AD neuropathologic changes (ADNC) and copathologies. In this study, we tested the hypothesis that the APOE alleles differentially affect the rate of cognitive decline at the normal aging-early AD continuum and that this association is independent of their effects on classical ADNC and copathologies.
Methods We analyzed APOE associations with the cognitive trajectories (Clinical Dementia Rating scale Sum of Boxes [CDR-SOB] and Mini-Mental State Examination [MMSE]) of more than 1,000 individuals from a national clinicopathologic sample who had either no, mild (sparse neuritic plaques and the Braak neurofibrillary tangle [NFT] stage I/II), or intermediate (moderate neuritic plaques and the Braak NFT stage III/IV) ADNC levels at autopsy via 2 latent classes reverse-time longitudinal modeling.
Results Carrying the APOEε4 allele was associated with a faster rate of cognitive decline by both CDR-SOB and MMSE relative to APOEε3 homozygotes. This association remained statistically significant after adjusting for ADNC severity, comorbid pathologies, and the effects of ADNC on the slope of cognitive decline. Our modeling strategy identified 2 latent classes in which APOEε4 carriers declined faster than APOEε3 homozygotes, with latent class 1 members representing slow decliners (CDR-SOB: 76.7\% of individuals, 0.195 vs 0.146 points/y in APOEε4 vs APOEε3/ε3; MMSE: 88.6\% of individuals, −0.303 vs −0.153 points/y in APOEε4 vs APOEε3/ε3), whereas latent class 2 members were fast decliners (CDR-SOB: 23.3\% of participants, 1.536 vs 1.487 points/y in APOEε4 vs APOEε3/ε3; MMSE: 11.4\% of participants, −2.538 vs −2.387 points/y in APOEε4 vs APOEε3/ε3). Compared with slow decliners, fast decliners were more likely to carry the APOEε4 allele, younger at initial visit and death, more impaired at initial and last visits, and more likely to have intermediate (vs none or mild) ADNC levels, as well as concurrent Lewy bodies and hippocampal sclerosis at autopsy.
Discussion In a large national sample selected to represent the normal aging-early AD continuum, the APOEε4 allele is associated with a modest but statistically significant acceleration of the cognitive decline rate even after controlling for its effects on ADNC and comorbid pathologies.},
	language = {en},
	number = {1},
	urldate = {2023-04-08},
	journal = {Neurology Genetics},
	author = {Qian, Jing and Zhang, Yiding and Betensky, Rebecca A. and Hyman, Bradley T. and Serrano-Pozo, Alberto},
	month = feb,
	year = {2023},
	pages = {e200055},
	note = {Publisher: Wolters Kluwer Health, Inc. on behalf of the American Academy of Neurology
Section: Research Article},
}

@misc{das_distinct_2023,
	title = {Distinct {Transcriptomic} {Responses} to {Aβ} plaques, {Neurofibrillary} {Tangles}, and {APOE} in {Alzheimer}’s {Disease}},
	copyright = {© 2023, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/},
	url = {https://www.biorxiv.org/content/10.1101/2023.03.20.533303v1},
	doi = {10.1101/2023.03.20.533303},
	abstract = {INTRODUCTION Omics studies have revealed that various brain cell types undergo profound molecular changes in Alzheimer’s disease (AD) but the spatial relationships with plaques and tangles and APOE-linked differences remain unclear.
METHODS We performed laser capture microdissection of Aβ plaques, the 50μm halo around them, tangles with the 50μm halo around them, and areas distant ({\textgreater}50μm) from plaques and tangles in the temporal cortex of AD and control donors, followed by RNA-sequencing.
RESULTS Aβ plaques exhibited upregulated microglial (neuroinflammation/phagocytosis) and downregulated neuronal (neurotransmission/energy metabolism) genes, whereas tangles had mostly downregulated neuronal genes. Aβ plaques had more differentially expressed genes than tangles. We identified a gradient Aβ plaque{\textgreater}peri-plaque{\textgreater}tangle{\textgreater}distant for these changes. AD APOEε4 homozygotes had greater changes than APOEε3 across locations, especially within Aβ plaques.
DISCUSSION Transcriptomic changes in AD consist primarily of neuroinflammation and neuronal dysfunction, are spatially associated mainly with Aβ plaques, and are exacerbated by the APOEε4 allele.},
	language = {en},
	urldate = {2023-04-08},
	publisher = {bioRxiv},
	author = {Das, Sudeshna and Li, Zhaozhi and Wachter, Astrid and Alla, Srinija and Noori, Ayush and Abdourahman, Aicha and Tamm, Joseph A. and Woodbury, Maya E. and Talanian, Robert V. and Biber, Knut and Karran, Eric H. and Hyman, Bradley T. and Serrano-Pozo, Alberto},
	month = mar,
	year = {2023},
	note = {bioRxiv 2023.03.20.533303},
}

@incollection{munoz-castro_cyclic_2023,
	address = {New York, NY},
	series = {Methods in {Molecular} {Biology}},
	title = {Cyclic {Multiplex} {Fluorescent} {Immunohistochemistry} {Protocol} to {Phenotype} {Glial} {Cells} in {Formalin}-{Fixed} {Paraffin}-{Embedded} {Human} {Brain} {Sections}},
	isbn = {978-1-07-162811-9},
	url = {https://doi.org/10.1007/978-1-0716-2811-9_19},
	abstract = {There is a growing interest in expanding the multiplexing capability of immunohistochemistry to achieve a deeper phenotyping of various cell types in health and disease. Here, we describe a protocol of cyclic multiplex fluorescent immunohistochemistry that enables the labeling of up to 16 antigens on the same formalin-fixed paraffin-embedded section using “off-the-shelf,” commercially available, primary antibodies as well as fluorescently conjugated secondary antibodies. Key steps include the denaturing/stripping of the antibodies by microwaving and the quenching of any remaining fluorescent signal between the cycles of otherwise traditional multiplexed fluorescent immunohistochemistry. We have successfully applied this protocol to characterize astrocytic and microglial responses to Aβ plaques and neurofibrillary tangles in Alzheimer’s disease brains, but it could be easily adapted to other user’s needs regarding cell types, disease, and organ.},
	language = {en},
	urldate = {2023-04-08},
	booktitle = {Signal {Transduction} {Immunohistochemistry}: {Methods} and {Protocols}},
	publisher = {Springer US},
	author = {Muñoz-Castro, Clara and Noori, Ayush and Hyman, Bradley T. and Serrano-Pozo, Alberto},
	editor = {Kalyuzhny, Alexander E.},
	year = {2023},
	doi = {10.1007/978-1-0716-2811-9_19},
	keywords = {Alignment, Amyloid-β plaques, Antibodies, Astrocytes, Cyclic multiplex fluorescence immunohistochemistry, Formalin-fixed paraffin-embedded sections, Glial cells, Microglia, Neurofibrillary tangles, Segmentation},
	pages = {283--305},
}

@article{munoz-castro_cyclic_2022,
	title = {Cyclic multiplex fluorescent immunohistochemistry and machine learning reveal distinct states of astrocytes and microglia in normal aging and {Alzheimer}’s disease},
	volume = {19},
	issn = {1742-2094},
	url = {https://doi.org/10.1186/s12974-022-02383-4},
	doi = {10.1186/s12974-022-02383-4},
	abstract = {Astrocytes and microglia react to Aβ plaques, neurofibrillary tangles, and neurodegeneration in the Alzheimer’s disease (AD) brain. Single-nuclei and single-cell RNA-seq have revealed multiple states or subpopulations of these glial cells but lack spatial information. We have developed a methodology of cyclic multiplex fluorescent immunohistochemistry on human postmortem brains and image analysis that enables a comprehensive morphological quantitative characterization of astrocytes and microglia in the context of their spatial relationships with plaques and tangles.},
	language = {en},
	number = {1},
	urldate = {2023-04-08},
	journal = {Journal of Neuroinflammation},
	author = {Muñoz-Castro, Clara and Noori, Ayush and Magdamo, Colin G. and Li, Zhaozhi and Marks, Jordan D. and Frosch, Matthew P. and Das, Sudeshna and Hyman, Bradley T. and Serrano-Pozo, Alberto},
	month = feb,
	year = {2022},
	keywords = {Alzheimer’s disease, Amyloid plaques, Astrocytes, Immunohistochemistry, Microglia, Neurofibrillary tangles, Neuropathology, Tau},
	pages = {30},
}

@article{viejo_systematic_2022,
	title = {Systematic review of human post-mortem immunohistochemical studies and bioinformatics analyses unveil the complexity of astrocyte reaction in {Alzheimer}'s disease},
	volume = {48},
	issn = {1365-2990},
	url = {https://onlinelibrary.wiley.com/doi/abs/10.1111/nan.12753},
	doi = {10.1111/nan.12753},
	abstract = {Aims Reactive astrocytes in Alzheimer's disease (AD) have traditionally been demonstrated by increased glial fibrillary acidic protein (GFAP) immunoreactivity; however, astrocyte reaction is a complex and heterogeneous phenomenon involving multiple astrocyte functions beyond cytoskeletal remodelling. To better understand astrocyte reaction in AD, we conducted a systematic review of astrocyte immunohistochemical studies in post-mortem AD brains followed by bioinformatics analyses on the extracted reactive astrocyte markers. Methods NCBI PubMed, APA PsycInfo and WoS-SCIE databases were interrogated for original English research articles with the search terms ‘Alzheimer's disease’ AND ‘astrocytes.’ Bioinformatics analyses included protein–protein interaction network analysis, pathway enrichment, and transcription factor enrichment, as well as comparison with public human -omics datasets. Results A total of 306 articles meeting eligibility criteria rendered 196 proteins, most of which were reported to be upregulated in AD vs control brains. Besides cytoskeletal remodelling (e.g., GFAP), bioinformatics analyses revealed a wide range of functional alterations including neuroinflammation (e.g., IL6, MAPK1/3/8 and TNF), oxidative stress and antioxidant defence (e.g., MT1A/2A, NFE2L2, NOS1/2/3, PRDX6 and SOD1/2), lipid metabolism (e.g., APOE, CLU and LRP1), proteostasis (e.g., cathepsins, CRYAB and HSPB1/2/6/8), extracellular matrix organisation (e.g., CD44, MMP1/3 and SERPINA3), and neurotransmission (e.g., CHRNA7, GABA, GLUL, GRM5, MAOB and SLC1A2), among others. CTCF and ESR1 emerged as potential transcription factors driving these changes. Comparison with published -omics datasets validated our results, demonstrating a significant overlap with reported transcriptomic and proteomic changes in AD brains and/or CSF. Conclusions Our systematic review of the neuropathological literature reveals the complexity of AD reactive astrogliosis. We have shared these findings as an online resource available at www.astrocyteatlas.org.},
	language = {en},
	number = {1},
	urldate = {2023-04-08},
	journal = {Neuropathology and Applied Neurobiology},
	author = {Viejo, Lucía and Noori, Ayush and Merrill, Emily and Das, Sudeshna and Hyman, Bradley T. and Serrano-Pozo, Alberto},
	year = {2022},
	note = {\_eprint: https://onlinelibrary.wiley.com/doi/pdf/10.1111/nan.12753},
	keywords = {Alzheimer's disease, astrocyte, bioinformatics, immunohistochemistry, neuropathology, reactive astrogliosis, systematic review},
	pages = {e12753},
}

@misc{serrano-pozo_astrocyte_2022,
	title = {Astrocyte transcriptomic changes along the spatiotemporal progression of {Alzheimer}’s disease},
	copyright = {© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/},
	url = {https://www.biorxiv.org/content/10.1101/2022.12.03.518999v1},
	doi = {10.1101/2022.12.03.518999},
	abstract = {Astrocytes play a critical role in brain homeostasis and normal functions but their changes along the spatiotemporal progression of Alzheimer’s disease (AD) neuropathology remain largely unknown. Here we performed single-nucleus RNA-sequencing on brain regions along the stereotypical progression of AD pathology from donors ranging the entire normal aging-AD continuum comprising 628,943 astrocyte nuclei from 32 donors across 5 brain regions. We discovered temporal gene-expression-trajectories with gene sets differentially activated at various disease stages. Surprisingly, a gene set enriched in proteostasis and energy metabolism, was upregulated in late-stage but unexpectedly returned to baseline levels in end-stage, suggesting exhaustion of response in “burnt-out” astrocytes. The spatial gene-expression-trajectories revealed that astrocytic genes of tripartite synapses are dysregulated in parallel to the stereotypical progression of tangle pathology across regions. We identified astrocyte heterogeneity across brain regions with a continuum from homeostatic to reactive cells through “intermediate” transitional states. These findings suggest complex astrocytic dysfunction in AD neurodegeneration.},
	language = {en},
	urldate = {2023-04-08},
	publisher = {bioRxiv},
	author = {Serrano-Pozo, Alberto and Li, Zhaozhi and Woodbury, Maya E. and Muñoz-Castro, Clara and Wachter, Astrid and Jayakumar, Rojashree and Bryant, Annie G. and Noori, Ayush and Welikovitch, Lindsay A. and Hu, Miwei and Zhao, Karen and Liao, Fan and Lin, Gen and Pastika, Timothy and Tamm, Joseph and Abdourahman, Aicha and Kwon, Taekyung and Bennett, Rachel E. and Talanian, Robert V. and Biber, Knut and Karran, Eric H. and Hyman, Bradley T. and Das, Sudeshna},
	month = dec,
	year = {2022},
	note = {bioRxiv 2022.12.03.518999},
}

@article{qian_association_2021,
	title = {Association of {APOE} {Genotype} {With} {Heterogeneity} of {Cognitive} {Decline} {Rate} in {Alzheimer} {Disease}},
	volume = {96},
	copyright = {Copyright © 2021 The Author(s). Published by Wolters Kluwer Health, Inc. on behalf of the American Academy of Neurology.},
	issn = {1526-632X 0028-3878},
	doi = {10.1212/WNL.0000000000011883},
	url = {https://doi.org/10.1212/WNL.0000000000011883},
	abstract = {OBJECTIVE: To test the hypothesis that the APOE genotype is a significant driver of heterogeneity in Alzheimer disease (AD) clinical progression, which could have  important implications for clinical trial design and interpretation. METHODS: We  applied novel reverse-time longitudinal models to analyze the trajectories of  Clinical Dementia Rating Sum of Boxes (CDR-SOB) and Mini-Mental State Examination  (MMSE) scores-2 common outcome measures in AD clinical trials-in 1,102  autopsy-proven AD cases (moderate/frequent neuritic plaques and Braak tangle stage  III or greater) from the National Alzheimer's Coordinating Center Neuropathology  database resembling participants with mild to moderate AD in therapeutic clinical  trials. RESULTS: APOE ε4 carriers exhibited ≈1.5 times faster CDR-SOB increase than  APOE ε3/ε3 carriers (2.12 points per year vs 1.44 points per year) and ≈1.3 times  faster increase than APOE ε2 carriers (1.65 points per year), whereas APOE ε2 vs  APOE ε3/ε3 difference was not statistically significant. APOE ε4 carriers had ≈1.1  times faster MMSE decline than APOE ε3/ε3 carriers (-3.45 vs -3.03 points per year)  and ≈1.4 times faster decline than APOE ε2 carriers (-2.43 points per year), whereas  APOE ε2 carriers had ≈1.2 times slower decline than APOE ε3/ε3 carriers (-2.43 vs  -3.03 points per year). These findings remained largely unchanged after controlling  for the effect of AD neuropathologic changes on the rate of cognitive decline and  for the presence and severity of comorbid pathologies. CONCLUSION: Compared to the  APOE ε3/ε3 reference genotype, the APOE ε2 and ε4 alleles have opposite (slowing and  accelerating, respectively) effects on the rate of cognitive decline, which are  clinically relevant and largely independent of the differential APOE allele effects  on AD and comorbid pathologies. Thus, APOE genotype contributes to the heterogeneity  in rate of clinical progression in AD.},
	language = {eng},
	number = {19},
	journal = {Neurology},
	author = {Qian, Jing and Betensky, Rebecca A. and Hyman, Bradley T. and Serrano-Pozo, Alberto},
	month = may,
	year = {2021},
	pmid = {33771840},
	pmcid = {PMC8166439},
	keywords = {*Disease Progression, *Genotype, Aged, Aged, 80 and over, Alzheimer Disease/diagnosis/epidemiology/*genetics, Apolipoprotein E2/*genetics, Apolipoprotein E3/genetics, Apolipoprotein E4/*genetics, Cognitive Dysfunction/diagnosis/epidemiology/*genetics, Cohort Studies, Female, Humans, Longitudinal Studies, Male},
	pages = {e2414--e2428},
}

@article{noori_differential_2021,
	title = {Differential gene expression data from the human central nervous system across {Alzheimer}'s disease, {Lewy} body diseases, and the amyotrophic lateral sclerosis and  frontotemporal dementia spectrum},
	volume = {35},
	copyright = {© 2021 The Authors. Published by Elsevier Inc.},
	issn = {2352-3409},
	doi = {10.1016/j.dib.2021.106863},
	url = {https://doi.org/10.1016/j.dib.2021.106863},
	abstract = {In Noori et al. [1], we hypothesized that there is a shared gene expression signature underlying neurodegenerative proteinopathies including Alzheimer's disease  (AD), Lewy body diseases (LBD), and the amyotrophic lateral sclerosis and  frontotemporal dementia (ALS-FTD) spectrum. To test this hypothesis, we performed a  systematic review and meta-analysis of 60 human central nervous system  transcriptomic datasets in the public Gene Expression Omnibus and ArrayExpress  repositories, comprising a total of 2,600 AD, LBD, and ALS-FTD patients and  age-matched controls which passed our stringent quality control pipeline. Here, we  provide the results of differential expression analyses with data quality reports  for each of these 60 datasets. This atlas of differential expression across AD, LBD,  and ALS-FTD may guide future work to elucidate the pathophysiological drivers of  these individual diseases as well as the common substrate of neurodegeneration.},
	language = {eng},
	journal = {Data Brief},
	author = {Noori, Ayush and Mezlini, Aziz M. and Hyman, Bradley T. and Serrano-Pozo, Alberto and Das, Sudeshna},
	month = apr,
	year = {2021},
	pmid = {33665258},
	pmcid = {PMC7903289},
	keywords = {Alzheimer's disease, Amyotrophic lateral sclerosis, Differential expression, Frontotemporal dementia, Lewy body diseases, Meta-analysis, Neurodegeneration, Transcriptomics},
	pages = {106863},
}

@article{escartin_reactive_2021,
	title = {Reactive astrocyte nomenclature, definitions, and future directions},
	volume = {24},
	issn = {1546-1726 1097-6256},
	doi = {10.1038/s41593-020-00783-4},
	url = {https://doi.org/10.1038/s41593-020-00783-4},
	abstract = {Reactive astrocytes are astrocytes undergoing morphological, molecular, and functional remodeling in response to injury, disease, or infection of the CNS.  Although this remodeling was first described over a century ago, uncertainties and  controversies remain regarding the contribution of reactive astrocytes to CNS  diseases, repair, and aging. It is also unclear whether fixed categories of reactive  astrocytes exist and, if so, how to identify them. We point out the shortcomings of  binary divisions of reactive astrocytes into good-vs-bad,  neurotoxic-vs-neuroprotective or A1-vs-A2. We advocate, instead, that research on  reactive astrocytes include assessment of multiple molecular and functional  parameters-preferably in vivo-plus multivariate statistics and determination of  impact on pathological hallmarks in relevant models. These guidelines may spur the  discovery of astrocyte-based biomarkers as well as astrocyte-targeting therapies  that abrogate detrimental actions of reactive astrocytes, potentiate their neuro-  and glioprotective actions, and restore or augment their homeostatic, modulatory,  and defensive functions.},
	language = {eng},
	number = {3},
	journal = {Nat Neurosci},
	author = {Escartin, Carole and Galea, Elena and Lakatos, András and O'Callaghan, James P. and Petzold, Gabor C. and Serrano-Pozo, Alberto and Steinhäuser, Christian and Volterra, Andrea and Carmignoto, Giorgio and Agarwal, Amit and Allen, Nicola J. and Araque, Alfonso and Barbeito, Luis and Barzilai, Ari and Bergles, Dwight E. and Bonvento, Gilles and Butt, Arthur M. and Chen, Wei-Ting and Cohen-Salmon, Martine and Cunningham, Colm and Deneen, Benjamin and De Strooper, Bart and Díaz-Castro, Blanca and Farina, Cinthia and Freeman, Marc and Gallo, Vittorio and Goldman, James E. and Goldman, Steven A. and Götz, Magdalena and Gutiérrez, Antonia and Haydon, Philip G. and Heiland, Dieter H. and Hol, Elly M. and Holt, Matthew G. and Iino, Masamitsu and Kastanenka, Ksenia V. and Kettenmann, Helmut and Khakh, Baljit S. and Koizumi, Schuichi and Lee, C. Justin and Liddelow, Shane A. and MacVicar, Brian A. and Magistretti, Pierre and Messing, Albee and Mishra, Anusha and Molofsky, Anna V. and Murai, Keith K. and Norris, Christopher M. and Okada, Seiji and Oliet, Stéphane H. R. and Oliveira, João F. and Panatier, Aude and Parpura, Vladimir and Pekna, Marcela and Pekny, Milos and Pellerin, Luc and Perea, Gertrudis and Pérez-Nievas, Beatriz G. and Pfrieger, Frank W. and Poskanzer, Kira E. and Quintana, Francisco J. and Ransohoff, Richard M. and Riquelme-Perez, Miriam and Robel, Stefanie and Rose, Christine R. and Rothstein, Jeffrey D. and Rouach, Nathalie and Rowitch, David H. and Semyanov, Alexey and Sirko, Swetlana and Sontheimer, Harald and Swanson, Raymond A. and Vitorica, Javier and Wanner, Ina-Beate and Wood, Levi B. and Wu, Jiaqian and Zheng, Binhai and Zimmer, Eduardo R. and Zorec, Robert and Sofroniew, Michael V. and Verkhratsky, Alexei},
	month = mar,
	year = {2021},
	pmid = {33589835},
	pmcid = {PMC8007081},
	keywords = {Aging/*pathology, Animals, Astrocytes/*pathology, Brain Diseases/pathology, Brain Injuries/pathology, Brain/*pathology, Humans, Spinal Cord Injuries/pathology, Spinal Cord/*pathology},
	pages = {312--325},
}

@article{noori_systematic_2021,
	title = {Systematic review and meta-analysis of human transcriptomics reveals neuroinflammation, deficient energy metabolism, and proteostasis failure across  neurodegeneration},
	volume = {149},
	copyright = {Copyright © 2020 The Authors. Published by Elsevier Inc. All rights reserved.},
	issn = {1095-953X 0969-9961},
	doi = {10.1016/j.nbd.2020.105225},
	url = {https://doi.org/10.1016/j.nbd.2020.105225},
	abstract = {Neurodegenerative disorders such as Alzheimer's disease (AD), Lewy body diseases (LBD), and the amyotrophic lateral sclerosis and frontotemporal dementia (ALS-FTD)  spectrum are defined by the accumulation of specific misfolded protein aggregates.  However, the mechanisms by which each proteinopathy leads to neurodegeneration  remain elusive. We hypothesized that there is a common "pan-neurodegenerative" gene  expression signature driving pathophysiology across these clinically and  pathologically diverse proteinopathies. To test this hypothesis, we performed a  systematic review of human CNS transcriptomics datasets from AD, LBD, and ALS-FTD  patients and age-matched controls in the Gene Expression Omnibus (GEO) and  ArrayExpress databases, followed by consistent processing of each dataset,  meta-analysis, pathway enrichment, and overlap analyses. After applying  pre-specified eligibility criteria and stringent data pre-processing, a total of  2600 samples from 26 AD, 21 LBD, and 13 ALS-FTD datasets were included in the  meta-analysis. The pan-neurodegenerative gene signature is characterized by an  upregulation of innate immunity, cytoskeleton, and transcription and RNA processing  genes, and a downregulation of the mitochondrial electron transport chain. Pathway  enrichment analyses also revealed the upregulation of neuroinflammation (including  Toll-like receptor, TNF, and NFκB signaling) and phagocytosis, and the  downregulation of mitochondrial oxidative phosphorylation, lysosomal acidification,  and ubiquitin-proteasome pathways. Our findings suggest that neuroinflammation and a  failure in both neuronal energy metabolism and protein degradation systems are  consistent features underlying neurodegenerative diseases, despite differences in  the extent of neuronal loss and brain regions involved.},
	language = {eng},
	journal = {Neurobiol Dis},
	author = {Noori, Ayush and Mezlini, Aziz M. and Hyman, Bradley T. and Serrano-Pozo, Alberto and Das, Sudeshna},
	month = feb,
	year = {2021},
	pmid = {33347974},
	pmcid = {PMC7856076},
	keywords = {*Alzheimer's disease, *Amyotrophic lateral sclerosis, *Frontotemporal dementia, *Lewy body diseases, *Meta-analysis, *Mitochondrial energy metabolism, *Neurodegeneration, *Neuroinflammation, *Proteostasis, *Transcriptomics},
	pages = {105225},
}

@article{serrano-pozo_apoe_2021,
	title = { {APOE} and {Alzheimer}'s disease: advances in genetics, pathophysiology, and therapeutic approaches},
	volume = {20},
	copyright = {Copyright © 2021 Elsevier Ltd. All rights reserved.},
	issn = {1474-4465 1474-4422},
	doi = {10.1016/S1474-4422(20)30412-9},
	url = {https://doi.org/10.1016/S1474-4422(20)30412-9},
	abstract = {The APOE ε4 allele remains the strongest genetic risk factor for sporadic Alzheimer's disease and the APOE ε2 allele the strongest genetic protective factor  after multiple large scale genome-wide association studies and genome-wide  association meta-analyses. However, no therapies directed at APOE are currently  available. Although initial studies causally linked APOE with amyloid-β peptide  aggregation and clearance, over the past 5 years our understanding of APOE  pathogenesis has expanded beyond amyloid-β peptide-centric mechanisms to tau  neurofibrillary degeneration, microglia and astrocyte responses, and blood-brain  barrier disruption. Because all these pathological processes can potentially  contribute to cognitive impairment, it is important to use this new knowledge to  develop therapies directed at APOE. Several therapeutic approaches have been  successful in mouse models expressing human APOE alleles, including increasing or  reducing APOE levels, enhancing its lipidation, blocking the interactions between  APOE and amyloid-β peptide, and genetically switching APOE4 to APOE3 or APOE2  isoforms, but translation to human clinical trials has proven challenging.},
	language = {eng},
	number = {1},
	journal = {Lancet Neurol},
	author = {Serrano-Pozo, Alberto and Das, Sudeshna and Hyman, Bradley T.},
	month = jan,
	year = {2021},
	pmid = {33340485},
	pmcid = {PMC8096522},
	keywords = {*Alzheimer Disease/drug therapy/genetics/metabolism/pathology, *Apolipoproteins E/drug effects/genetics/metabolism, *Genetic Therapy, Animals, Humans},
	pages = {68--80},
}

@article{das_meta-analysis_2020,
	title = {Meta-analysis of mouse transcriptomic studies supports a context-dependent astrocyte reaction in acute {CNS} injury versus neurodegeneration},
	volume = {17},
	issn = {1742-2094},
	doi = {10.1186/s12974-020-01898-y},
	url = {https://doi.org/10.1186/s12974-020-01898-y},
	abstract = {BACKGROUND: Neuronal damage in acute CNS injuries and chronic neurodegenerative diseases is invariably accompanied by an astrocyte reaction in both mice and humans.  However, whether and how the nature of the CNS insult-acute versus  chronic-influences the astrocyte response, and whether astrocyte transcriptomic  changes in these mouse models faithfully recapitulate the astrocyte reaction in  human diseases remains to be elucidated. We hypothesized that astrocytes set off  different transcriptomic programs in response to acute versus chronic insults,  besides a shared "pan-injury" signature common to both types of conditions, and  investigated the presence of these mouse astrocyte signatures in transcriptomic  studies from human neurodegenerative diseases. METHODS: We performed a meta-analysis  of 15 published astrocyte transcriptomic datasets from mouse models of acute injury  (n = 6) and chronic neurodegeneration (n = 9) and identified pan-injury, acute, and  chronic signatures, with both upregulated (UP) and downregulated (DOWN) genes. Next,  we investigated these signatures in 7 transcriptomic datasets from various human  neurodegenerative diseases. RESULTS: In mouse models, the number of UP/DOWN genes  per signature was 64/21 for pan-injury and 109/79 for acute injury, whereas only  13/27 for chronic neurodegeneration. The pan-injury-UP signature was represented by  the classic cytoskeletal hallmarks of astrocyte reaction (Gfap and Vim), plus  extracellular matrix (i.e., Cd44, Lgals1, Lgals3, Timp1), and immune response (i.e.,  C3, Serping1, Fas, Stat1, Stat2, Stat3). The acute injury-UP signature was enriched  in protein synthesis and degradation (both ubiquitin-proteasome and autophagy  systems), intracellular trafficking, and anti-oxidant defense genes, whereas the  acute injury-DOWN signature included genes that regulate chromatin structure and  transcriptional activity, many of which are transcriptional repressors. The chronic  neurodegeneration-UP signature was further enriched in astrocyte-secreted  extracellular matrix proteins (Lama4, Cyr61, Thbs4), while the DOWN signature  included relevant genes such as Agl (glycogenolysis), S1pr1 (immune modulation), and  Sod2 (anti-oxidant). Only the pan-injury-UP mouse signature was clearly present in  some human neurodegenerative transcriptomic datasets. CONCLUSIONS: Acute and chronic  CNS injuries lead to distinct astrocyte gene expression programs beyond their common  astrocyte reaction signature. However, caution should be taken when extrapolating  astrocyte transcriptomic findings from mouse models to human diseases.},
	language = {eng},
	number = {1},
	journal = {J Neuroinflammation},
	author = {Das, Sudeshna and Li, Zhaozhi and Noori, Ayush and Hyman, Bradley T. and Serrano-Pozo, Alberto},
	month = jul,
	year = {2020},
	pmid = {32736565},
	pmcid = {PMC7393869},
	keywords = {*Transcriptome, Acute CNS injury, Animals, Astrocyte reaction, Astrocytes/*metabolism, Gene Expression Profiling, Glial Fibrillary Acidic Protein/metabolism, Meta-analysis, Mice, Nerve Degeneration/genetics/*metabolism/pathology, Neurodegenerative diseases, Neurons/*metabolism/pathology, Transcriptomics},
	pages = {227},
}

@article{gui_characterization_2020,
	title = {Characterization of the 18 {kDa} translocator protein ({TSPO}) expression in post-mortem normal and {Alzheimer}'s disease brains},
	volume = {30},
	copyright = {© 2019 The Authors. Brain Pathology published by John Wiley \& Sons Ltd on behalf of International Society of Neuropathology.},
	issn = {1750-3639 1015-6305},
	doi = {10.1111/bpa.12763},
	url = {https://doi.org/10.1111/bpa.12763},
	abstract = {The 18 kDa translocator protein (TSPO) is a widely used target for microglial PET imaging radioligands, but its expression in post-mortem normal and diseased human  brain is not well described. We aimed at characterizing the TSPO expression in human  control (CTRL) and Alzheimer's disease (AD) brains. Specifically, we sought to: (1)  define the cell type(s) expressing TSPO; (2) compare tspo mRNA and TSPO levels  between AD and CTRL brains; (3) correlate TSPO levels with quantitative  neuropathological measures of reactive glia and AD neuropathological changes; and  (4) investigate the effects of the TSPO rs6971 SNP on tspo mRNA and TSPO levels,  glial responses and AD neuropathological changes. We performed quantitative  immunohistochemistry and Western blot in post-mortem brain samples from CTRL and AD  subjects, as well as analysis of publicly available mouse and human brain RNA-Seq  datasets. We found that: (1) TSPO is expressed not just in microglia, but also in  astrocytes, endothelial cells and vascular smooth muscle cells; (2) there is  substantial overlap of tspo mRNA and TSPO levels between AD and CTRL subjects and in  TSPO levels between temporal neocortex and white matter in both groups; (3) TSPO  cortical burden does not correlate with the burden of activated microglia or  reactive astrocytes, Aβ plaques or neurofibrillary tangles, or the cortical  thickness; (4) the TSPO rs6971 SNP does not significantly impact tspo mRNA or TSPO  levels, the magnitude of glial responses, the cortical thickness, or the burden of  AD neuropathological changes. These results could inform ongoing efforts toward the  development of reactive glia-specific PET radioligands.},
	language = {eng},
	number = {1},
	journal = {Brain Pathol},
	author = {Gui, Yaxing and Marks, Jordan D. and Das, Sudeshna and Hyman, Bradley T. and Serrano-Pozo, Alberto},
	month = jan,
	year = {2020},
	pmid = {31276244},
	pmcid = {PMC6904423},
	keywords = {*18 kDa translocator protein (TSPO), *Alzheimer's disease, *amyloid plaques, *astrocytes, *microglia, *neurofibrillary tangles, *neuroinflammation, *peripheral benzodiazepine receptor, *positron emission tomography (PET), *rs6971 SNP, Alzheimer Disease/*genetics/pathology, Astrocytes/pathology, Autopsy/methods, Brain/pathology, Endothelial Cells/pathology, Gene Expression/genetics, Humans, Longitudinal Studies, Microglia/pathology, Neurofibrillary Tangles/metabolism, Neuroglia/pathology, Plaque, Amyloid/pathology, Positron-Emission Tomography, Receptors, GABA/*genetics/*metabolism, Transcriptome/genetics},
	pages = {151--164},
}

@article{serrano-pozo_is_2019,
	title = {Is {Alzheimer}'s {Disease} {Risk} {Modifiable}?},
	volume = {67},
	issn = {1875-8908 1387-2877},
	doi = {10.3233/JAD181028},
	url = {https://doi.org/10.3233/JAD181028},
	abstract = {Population-based clinic-pathological studies have established that the most common pathological substrate of dementia in community-dwelling elderly people is mixed,  especially Alzheimer's disease (AD) and cerebrovascular ischemic disease (CVID),  rather than pure AD. While these could be just two frequent unrelated comorbidities  in the elderly, epidemiological research has reinforced the idea that mid-life (age  {\textless}65 years) vascular risk factors increase the risk of late-onset (age ≥ 65 years)  dementia, and specifically AD. By contrast, healthy lifestyle choices such as  leisure activities, physical exercise, and Mediterranean diet are considered  protective against AD. Remarkably, several large population-based longitudinal  epidemiological studies have recently indicated that the incidence and prevalence of  dementia might be decreasing in Western countries. Although it remains unclear  whether these positive trends are attributable to neuropathologically definite AD  versus CVID, based on these epidemiological data it has been estimated that a  sizable proportion of AD cases could be preventable. In this review, we discuss the  current evidence about modifiable risk factors for AD derived from epidemiological,  preclinical, and interventional studies, and analyze the opportunities for  therapeutic and preventative interventions.},
	language = {eng},
	number = {3},
	journal = {J Alzheimers Dis},
	author = {Serrano-Pozo, Alberto and Growdon, John H.},
	year = {2019},
	pmid = {30776012},
	pmcid = {PMC6708279},
	keywords = {*Alcohol drinking, *Alzheimer’s disease, *dementia, *diabetes mellitus, *diet, *education, *exercise, *hyperlipidemia, *hypertension, *smoking, Alzheimer Disease/epidemiology/etiology/*prevention \& control, Genetic Predisposition to Disease/genetics, Humans, Prevalence, Risk Factors},
	pages = {795--819},
}

@article{perez-nievas_editorial_2018,
	title = {Editorial: {The} {Role} of {Glia} in {Alzheimer}'s {Disease}},
	volume = {9},
	issn = {1664-2295},
	doi = {10.3389/fneur.2018.01161},
	url = {https://doi.org/10.3389/fneur.2018.01161},
	language = {eng},
	journal = {Front Neurol},
	author = {Pérez-Nievas, Beatriz G. and Serrano-Pozo, Alberto},
	year = {2018},
	pmid = {30692961},
	pmcid = {PMC6340091},
	keywords = {Alzheimer's disease, amyloid plaques, astrocytes, chemokines/chemokine receptors, glia, microglia, neurofibrillary tangles (NFTs), neuroinflammation},
	pages = {1161},
}

@article{perez-nievas_deciphering_2018,
	title = {Deciphering the {Astrocyte} {Reaction} in {Alzheimer}'s {Disease}},
	volume = {10},
	issn = {1663-4365},
	doi = {10.3389/fnagi.2018.00114},
	url = {https://doi.org/10.3389/fnagi.2018.00114},
	abstract = {Reactive astrocytes were identified as a component of senile amyloid plaques in the cortex of Alzheimer's disease (AD) patients several decades ago. However, their role  in AD pathophysiology has remained elusive ever since, in part owing to the  extrapolation of the literature from primary astrocyte cultures and acute brain  injury models to a chronic neurodegenerative scenario. Recent accumulating evidence  supports the idea that reactive astrocytes in AD acquire neurotoxic properties,  likely due to both a gain of toxic function and a loss of their neurotrophic  effects. However, the diversity and complexity of this glial cell is only beginning  to be unveiled, anticipating that astrocyte reaction might be heterogeneous as well.  Herein we review the evidence from mouse models of AD and human neuropathological  studies and attempt to decipher the main conundrums that astrocytes pose to our  understanding of AD development and progression. We discuss the morphological  features that characterize astrocyte reaction in the AD brain, the consequences of  astrocyte reaction for both astrocyte biology and AD pathological hallmarks, and the  molecular pathways that have been implicated in this reaction.},
	language = {eng},
	journal = {Front Aging Neurosci},
	author = {Perez-Nievas, Beatriz G. and Serrano-Pozo, Alberto},
	year = {2018},
	pmid = {29922147},
	pmcid = {PMC5996928},
	keywords = {Alzheimer’s disease, amyloid plaques, astrocytes, glia, microglia, neurofibrillary tangles},
	pages = {114},
}

									</pre>
								</div>
							</div>

					</section>

				<!-- Footer -->
					<section id="footer">
						<div class="inner">
							<h2 class="major" id="contact">Contact</h2>
							<p>Interested in our work? Please don't hesitate to reach out. We look forward to hearing from you.</p>
							<ul class="contact">
								<li class="icon solid fa-home">
									MassGeneral Institute for Neurodegenerative Disease<br/>
									114 16th Street, Suite 2012<br/>
									Charlestown, MA 02129
								</li>
								<li class="icon solid fa-phone">(617) 643-0870</li>
								<li class="icon solid fa-envelope"><a href="mailto:aserrano1@mgh.harvard.edu">aserrano1@mgh.harvard.edu</a></li>
							</ul>
							<ul class="copyright">
								<li>Website developed by Ayush Noori.</li>
								<li style="float: right; border-left: none; padding-left: 0em;"><a href="https://www.massgeneral.org/" class="image" target="_blank" rel="noopener noreferrer"><img src="images/MGH_logo.svg" style="height: 6em;" alt=""/></a></li>
								<li style="float: right; border-left: none; padding-left: 0em;"><a href="https://hms.harvard.edu/" class="image" target="_blank" rel="noopener noreferrer"><img src="images/HMS_logo.svg" style="height: 6em;" alt=""/></a></li>
							</ul>
						</div>
					</section>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>
			<script src="assets/js/bib-list-min.js"></script>
			<script type="text/javascript">
				$(document).ready(function() {
				bibtexify("#bibtex", "pubTable");
				});
			</script>

	</body>
</html>